src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/StreamingSwap.tsx

```diff
@@ -21,12 +22,16 @@ export const StreamingSwap = (props: StreamingSwapProps) => {
     activeTradeId,
   )
 
+  const isInitializing = useMemo(() => {
+    return !isComplete && totalSwapCount === 0
+  }, [isComplete, totalSwapCount])
+
   return (
     <Stack px={4}>
       <Row>
         <Row.Label>{translate('trade.streamStatus')}</Row.Label>
         {totalSwapCount > 0 && (
-          <Row.Value>{`${attemptedSwapCount} of ${totalSwapCount}`}</Row.Value>
+          <Row.Value>{`${attemptedSwapCount - failedSwaps.length} of ${totalSwapCount}`}</Row.Value>
```

Comment:
🧠

Reply:
I'm actually not sure this is correct after looking again - the totalSwapCount probably includes failures too. Do tldr i should revert this change

![image](https://github.com/user-attachments/assets/a225831a-264c-4b30-9d9b-de8f7a589baa)

Reply:
7c5d92c35da443be6d6ea477933f24e7da85e64c